test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
> You nailed it! Our router plans are heavily inspired by Ember's routing system.

Ember's *old* router style. They dropped the state-machine API some time ago, because it was confusing and difficult for people that aren't intimately familiar with classic state-machine syntax. The current Ember router looks more like Rails' router.